code/modules/http/forumuser_api.dm

```diff
@@ -18,6 +19,8 @@ var/global/forumuser_api_key = null
 
 	discord_id = data["discord_id"]
 	ckey = data["ckey"]
+	if("psync_game_disalbed" in data && data["psync_game_disalbed"] == 1) //This here is needed because the data can be null, 1 or 0
```

Comment:
```suggestion
	if(data["psync_game_disalbed"] == 1) //This here is needed because the data can be null, 1 or 0
```

Precedent mistake, and the in check isn't necessary, as if the key is missing, it'll evaluate to null and thus false.